Modix/Modules/LmgtfyModule.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Net;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Discord;
+using Discord.Commands;
+
+namespace Modix.Modules
+{
+    [Name("Let Me Google That For You")]
+    [Summary("Sometimes we all need that little extra push")]
+    public sealed class LmgtfyModule : ModuleBase
+    {
+        private static readonly string[] WordChoices =
+        {
+            "your highness",
+            "your majesty",
+            "your excellency",
+            "my liege",
+            "o' great one",
+            "shatterer of a thousand worlds",
+            "dark lord",
+            "prince of thunder",
+            "great bambino",
+            "sultan of swat",
+            "titan of terror",
+            "colossus of clout"
+        };
+
+        private static readonly Random Random = new Random();
+
+        [Command("lmgtfy")]
+        [Summary("Creates a lmgtfy.com link")]
+        public async Task LmgtfyAsync([Remainder] string query)
```

Comment:
For completeness' sake, you should probably add a `[Summary]` to the parameter. Most other commands do.